server/tests/phase.service.test.js

```diff
@@ -64,4 +66,102 @@ describe('Phase Allocation Endpoints', () => {
     const res = await getAllSitePhases(site.id);
     expect(res.length).not.toEqual(0);
   });
+
+  it('getAllSitePhases, returns correct # of hires and remaining hires per phase', async () => {
+    const numAllocations = 30;
+    const phaseMockYearOne = {
+      name: '2015 Phase',
+      start_date: '2015/01/01',
+      end_date: '2015/12/31',
+    };
+    const phaseMockYearTwo = {
+      name: '2016 Phase',
+      start_date: '2016/01/01',
+      end_date: '2016/12/31',
+    };
+    const phaseYearOne = await createPhase(phaseMockYearOne, user);
+    const phaseYearTwo = await createPhase(phaseMockYearTwo, user);
+
+    const siteMock = siteData({
+      siteId: 7,
+      siteName: 'Test phase',
+      operatorEmail: 'test.e2e.phase@hcap.io',
+    });
+    const site = await makeTestSite(siteMock);
+
+    // An allocation for each created phase
+    await createAllocation({
+      site_id: site.id,
+      phase_id: phaseYearOne.id,
+      allocation: numAllocations,
+    }, user);
+    await createAllocation({
+      site_id: site.id,
```

Comment:
Prettier is throwing all sorts of spacing errors around here. Make sure to check your formatting.